v5/core/base_service_test.go

```diff
@@ -252,7 +252,7 @@ func TestRequestGoodResponseText(t *testing.T) {
 	assert.Nil(t, err)
 	assert.NotNil(t, service.Options.Authenticator)
 	assert.Equal(t, AUTHTYPE_NOAUTH, service.Options.Authenticator.AuthenticationType())
-	detailedResponse, err := service.Request(req, new(string))
+	detailedResponse, err := service.Request(req, new([]byte))
```

Comment:
I think for a text response, we'd expect the operation's return value to be `*string` (a string ptr).
```suggestion
        var result *string
	detailedResponse, err := service.Request(req, &result)
```
then we should probably verify with an assert that we got back the expected response string.
Looks like the test might have been at least slightly incorrect to begin with.

Reply:
If we make this change, then this test becomes redundant with the following test, `TestRequestGoodResponseString` -- both return a string response with `text/plain` content-type, and only differ in the type of `result`.  I accepted your suggestion, but now I think I will revert back to my original code unless you think I've missed something here.